chipsec/cfg/1022/strix.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+<configuration platform="Renoir">
```

Comment:
The platform attribute on the configuration tag needs to match the code attributes on the sku tag.